Sources/Sentry/SentrySDK.m

```diff
@@ -240,6 +240,7 @@ + (void)startWithOptions:(SentryOptions *)options
     SentryClient *newClient = [[SentryClient alloc] initWithOptions:options];
     [newClient.fileManager moveAppStateToPreviousAppState];
     [newClient.fileManager moveBreadcrumbsToPreviousBreadcrumbs];
+    [newClient.fileManager moveContextFileToPreviousContextFile];
```

Comment:
`m`: This adds another fileIO operation on the calling thread when starting the SDK. I know we simply follow the same approach, but I think we should consider moving all these operations do a BG thread, but not in this PR. WDTY?

Reply:
I don't think we can do this, because it should be a synchronous call during start so that the `contextFilePath` is ready to be used by the SDK. If we do this async, it could happen that the file is overwritten with new data, before it is moved to the `previousContextFilePath`.

Reply:
We don't have to fix this now. DispatchQueues are serial by default. We can move parts of these operations to a BG thread, when all of these run on the same serial DispatchQueue it should be doable.